codebuild/spec/buildspec_ubuntu_integrationv2.yml

```diff
@@ -55,12 +44,8 @@ phases:
         fi
       - $CB_BIN_DIR/install_default_dependencies.sh
       - $CB_BIN_DIR/s2n_setup_env.sh
-      - printenv
-      - make
-      # Install the parent s2n commit in a temporary directory
-      - $CB_BIN_DIR/install_s2n_head.sh "$(mktemp -d)"
```

Comment:
Not finding where this was moved to, did it get dropped or replaced?

Reply:
I removed the gcc part because it's already installed in install_ubuntu_dependencies.sh:
https://github.com/aws/s2n-tls/blob/f07656c07789987eefbc58244272a66a607a77bb/codebuild/bin/install_ubuntu_dependencies.sh#L31

Now that I call s2n_codebuild.sh, we no longer need make here:
https://github.com/aws/s2n-tls/blob/f07656c07789987eefbc58244272a66a607a77bb/codebuild/spec/buildspec_ubuntu_integrationv2.yml#L51

And I moved install_s2n_head.sh to s2n_codebuild.sh as well:
https://github.com/aws/s2n-tls/blob/f07656c07789987eefbc58244272a66a607a77bb/codebuild/bin/s2n_codebuild.sh#L76